django_orm_views/__init__.py

```diff
@@ -1,12 +1 @@
-from .views import (
-    PostgresMaterialisedViewMixin,
-    PostgresViewFromQueryset,
-    PostgresViewFromSQL,
-)
-
-from .sync import (
-    sync_views,
-    topological_sort_views,
-)
-
 default_app_config = 'django_orm_views.apps.DjangoPostgresViewsConfig'
```

Comment:
Is it possible to re-add these now?  It would be a shame if we can't just expose the interface directly in a flattened way

Reply:
The reason why I removed it was because I wanted to have all the views classes including readable views in the `views.py` file
If we keep `sync_views` and other imports in the  `__init__.py` then readable views get imported too early and that leads to an error

If we put readable views classes in a separate file then we can have those imports in the __init__.py

Should we separate them?